networkx/linalg/laplacianmatrix.py

```diff
@@ -46,6 +46,21 @@ def laplacian_matrix(G, nodelist=None, weight="weight"):
     to_numpy_array
     normalized_laplacian_matrix
     laplacian_spectrum
+
+    Examples
+    --------
+    For graphs with multiple connected components, L is permutation-similar
+    to a block diagonal matrix where each block is the respective Laplacian
+    matrix for each component.
+
+    >>> G = nx.graph_atlas(26) #This graph from the Graph Atlas has 2 connected components.
+    >>> print(nx.laplacian_matrix(G).todense())
```

Comment:
Using `toarray` would be preferable to `todense` here - we can also then get rid of the `print` and just rely on the array repr.

Reply:
great!

Reply:
> Using `toarray` would be preferable to `todense` here - we can also then get rid of the `print` and just rely on the array repr.

Without the `print` there are some docs tests that failed due to differences in some OS. On Windows at the end of the array is added `dtype=int32)`  but not on Ubuntu. Should I add the print back?

Reply:
Interesting - I wasn't expecting that but I suppose makes sense given windows integer types.

I suppose the two options are to add the `print` back or add an explicit cast, e.g. `toarray().astype(np.int64)`. Neither is ideal but both should be fine, so whatever you think is best!

Reply:
The platform-specific printing is likely related to something in numpy itself, see numpy/numpy#22586.

In the meantime, the printing workaround seems fine to me!